src/main/java/de/tum/in/www1/artemis/service/ComplaintResponseService.java

```diff
@@ -240,7 +240,7 @@ public boolean blockedByLock(ComplaintResponse complaintResponseRepresentingLock
      * @return true if the tutor is allowed to respond to the complaint, false otherwise
      */
     public boolean isUserAuthorizedToRespondToComplaint(Complaint complaint, User user) {
-        if (user == null || complaint == null || complaint.getResult() == null || complaint.getResult().getAssessor() == null) {
+        if (user == null || complaint == null || complaint.getResult() == null) {
```

Comment:
Why is the last check `complaint.getResult().getAssessor() == null` ommited?

Reply:
With the new assessment type it is now possible that a student complains for an automatic assessment that does not have an assessor.